content/en/docs/faq/issues.md

```diff
@@ -460,6 +460,32 @@ Delete the local git `jenkins-x-boot-config` repository.
 
 That should leave your Kubernetes provider and your local environment in a clean state.
 
+## No External IP assigned to the ingress in EKS
+
+If you have never created an Elastic Load Balancer (ELB) in AWS, then the `jx boot` command will fail to assign an ip
```

Comment:
Do you mind putting this into a single line? We try to aim for line per sentence since that makes diffing changes easier.

Reply:
@ankitm123 - Is this specific to using a custom domain or even an issue when using nip.io? I have not seen this myself, just wondering. Might be worth clarifying. 

Do you know what the underlying reason is for that? Is there a bug somewhere on our side or is this an AWS thing?

@dgozalo - do you know more about this?

Reply:
From what I know, this is on AWS.

Reply:
> Do you mind putting this into a single line? We try to aim for line per sentence since that makes diffing changes easier.

Done 👍

Reply:
> @ankitm123 - Is this specific to using a custom domain or even an issue when using nip.io? I have not seen this myself, just wondering. Might be worth clarifying.

I did not use custom domain, but I have seen others using custom domain run into this issue. Example: https://github.com/jenkins-x/jx/issues/7150
 
> Do you know what the underlying reason is for that? Is there a bug somewhere on our side or is this an AWS thing?
> 

This is an AWS issue. There is some relevant information here: https://github.com/terraform-aws-modules/terraform-aws-eks/issues/183#issuecomment-435315065

Reply:
Also this from AWS: https://eksworkshop.com/beginner/050_deploy/servicerole/, weird ...

Reply:
Actually think about it a bit more and looking at some other issues (https://github.com/terraform-aws-modules/terraform-aws-eks/issues/87), not sure how much of a value would it bring to include the additional burden of creating a service link role for elb. It can just be documented that a service link role for ELB should be created before provisioning EKS. Even AWS does the same thing in their eks workshop.

Reply:
> A fix would be to add the creation of this role in the jx aws eks module. I can test it out in my test aws account, and if it works, I can create a PR. @hferentschik wdyt?

+1,  I always prefer to fix the root cause :-)

Reply:
@ankitm123, there is also some discussion around missing tags!? Quite confusing everything.

Reply:
@ankitm123, what do you think we should do here now?

Reply:
I think we should document it, this is a a case that should not be handled by the eks module anyways. Even AWS in their EKS workshops makes it a pre-req for people to create a service role for ELB before creating an EKS cluster in a new AWS account. I feel adding the ability to create this role in jx terraform module just creates more maintenance burden, given the fact this only happens for new AWS accounts, with no ELBs created before.  

Even the upstream EKS module added it, and then reverted it back (https://github.com/terraform-aws-modules/terraform-aws-eks/issues/87#issuecomment-461119382). All in all, seems like documenting this is much better than making a code change. @hferentschik, thoughts? I think AWS might fix it in the future, seems more like a cloud provider issue ...